libs/SalesforceSDKCore/SalesforceSDKCore.xcodeproj/project.pbxproj

```diff
@@ -3011,6 +3032,7 @@
 				CLANG_WARN_ENUM_CONVERSION = YES;
 				CLANG_WARN_INT_CONVERSION = YES;
 				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
+				ENABLE_BITCODE = NO;
```

Comment:
Why no `Bitcode`?

Reply:
Compiler complains when building tests to run on device:
```
Showing All Messages
ld: -bundle and -bitcode_bundle (Xcode setting ENABLE_BITCODE=YES) cannot be used together
```
So either ENABLE_BITCODE has to be set to NO or the Match-O type for SalesforceSDKCoreTests target has to be changed to something else than Bundle.

Reply:
Actually disabling bitcode seems to be the only way to run it on device.
> ld: ‘/Applications/Xcode10.2.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library/Frameworks/XCTest.framework/XCTest’ does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. file ‘/Applications/Xcode10.2.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library/Frameworks/XCTest.framework/XCTest’ for architecture arm64